From 4377d51ef77152d798c7c7739a6e64c52911286c Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@cloudera.com>
Date: Thu, 3 Jun 2010 20:32:45 -0700
Subject: [PATCH 05/30] Re-enable log split test

---
 .../hbase/regionserver/wal/TestHLogSplit.java      |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/test/java/org/apache/hadoop/hbase/regionserver/wal/TestHLogSplit.java b/src/test/java/org/apache/hadoop/hbase/regionserver/wal/TestHLogSplit.java
index 9cbebb0..d7bdc44 100644
--- a/src/test/java/org/apache/hadoop/hbase/regionserver/wal/TestHLogSplit.java
+++ b/src/test/java/org/apache/hadoop/hbase/regionserver/wal/TestHLogSplit.java
@@ -404,7 +404,7 @@ public class TestHLogSplit {
       // hadoop 0.21 throws FNFE whereas hadoop 0.20 returns null
     }
   }
-/* DISABLED for now.  TODO: HBASE-2645 
+
   @Test
   public void testLogCannotBeWrittenOnceParsed() throws IOException {
     AtomicLong counter = new AtomicLong(0);
@@ -433,7 +433,7 @@ public class TestHLogSplit {
       stop.set(true);
     }
   }
-*/
+
 
   @Test
   public void testSplitWillNotTouchLogsIfNewHLogGetsCreatedAfterSplitStarted()
-- 
1.7.0.4

