From 89b8bdcb7eb2837c07a03783f2065be9f69a478d Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@lipcon.org>
Date: Sun, 20 Jun 2010 20:25:13 -0700
Subject: [PATCH 06/30] HBASE-2773. Check for null values in meta in test util

---
 .../apache/hadoop/hbase/HBaseTestingUtility.java   |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java b/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java
index fadee21..b0f132a 100644
--- a/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java
+++ b/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java
@@ -662,8 +662,10 @@ public class HBaseTestingUtility {
     List<byte[]> rows = new ArrayList<byte[]>();
     ResultScanner s = t.getScanner(new Scan());
     for (Result result : s) {
-      HRegionInfo info = Writables.getHRegionInfo(
-          result.getValue(HConstants.CATALOG_FAMILY, HConstants.REGIONINFO_QUALIFIER));
+      byte[] value = result.getValue(
+          HConstants.CATALOG_FAMILY, HConstants.REGIONINFO_QUALIFIER);
+      if (value == null) continue;
+      HRegionInfo info = Writables.getHRegionInfo(value);
       HTableDescriptor desc = info.getTableDesc();
       if (Bytes.compareTo(desc.getName(), tableName) == 0) {
         LOG.info("getMetaTableRows: row -> " +
-- 
1.7.0.4

