From 677ce02c0b4af1e361942e58bd8a34902cf363ca Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@cloudera.com>
Date: Wed, 15 Sep 2010 01:37:37 -0400
Subject: [PATCH 14/30] HBASE-3000. Add hbase classpath command

---
 bin/hbase |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/bin/hbase b/bin/hbase
index c783e80..ec92f7b 100755
--- a/bin/hbase
+++ b/bin/hbase
@@ -262,6 +262,9 @@ elif [ "$COMMAND" = "zookeeper" ] ; then
   fi
 elif [ "$COMMAND" = "zkcli" ] ; then
   CLASS='org.apache.zookeeper.ZooKeeperMain'
+elif [ "$COMMAND" = "classpath" ] ; then
+  echo $CLASSPATH
+  exit 0
 else
   CLASS=$COMMAND
 fi
-- 
1.7.0.4

