From 4c89df680a254d41fff96a4387280a8be505a2ad Mon Sep 17 00:00:00 2001
From: Todd Lipcon <todd@cloudera.com>
Date: Thu, 30 Sep 2010 22:42:07 -0700
Subject: [PATCH 18/30] CLOUDERA-BUILD. Build site as part of release build

---
 cloudera/do-release-build |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cloudera/do-release-build b/cloudera/do-release-build
index 0986c23..eacaf57 100755
--- a/cloudera/do-release-build
+++ b/cloudera/do-release-build
@@ -12,7 +12,7 @@ RELEASE_DIR=$BIN_DIR/..
 cd $RELEASE_DIR
 
 mvn -DskipTests clean
-mvn -DskipTests -Dhbase.version=${FULL_VERSION} install assembly:assembly 
+mvn -DskipTests -Dhbase.version=${FULL_VERSION} site install assembly:assembly 
 mkdir -p build
 for x in target/*.tar.gz ; do
   tar -C build -xzf $x
-- 
1.7.0.4

